packages/commercetools/helpers/src/index.ts

```diff
@@ -28,6 +30,23 @@ export const getProductVariants = (products: ProductVariant[], options: any = {}
   return products
 }
 
+export const getProductAttributes = (product: ProductVariant, filterByAttributeName?: Array<string>): any => {
```

Comment:
I guess your intention was `AgnosticProductAttribute` instead of `any`

Reply:
yup